pandas/io/json/json.py

```diff
@@ -349,6 +386,13 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
         lines = list(StringIO(json.strip()))
         json = '[' + ','.join(lines) + ']'
 
+    return _get_obj(typ, json, orient, dtype, convert_axes, convert_dates,
+                    keep_default_dates, numpy, precise_float, date_unit)
+
+
+def _get_obj(typ, json, orient, dtype, convert_axes, convert_dates,
+             keep_default_dates, numpy, precise_float,
+             date_unit):
```

Comment:
Why did you do this?  I like abstraction but not sure how this relates to your PR.